mobile/Features/CheckIn/AddMood.jsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import {
+  Text, View, TextInput, Pressable, TouchableWithoutFeedback,
+  Keyboard,
+  LogBox,
+} from 'react-native';
+import { useRoute } from '@react-navigation/native';
+import ProgressBar from 'react-native-progress/Bar';
+import PropTypes from 'prop-types';
+import styles from './MoodStyle';
+
+LogBox.ignoreLogs([
+  'Non-serializable values were found in the navigation state',
+]);
+
+function AddMood({ navigation }) {
+  // get parameters from route
+  const route = useRoute();
+  const setAddedMoods = route.params?.setAddedMoods;
```

Comment:
heyyyyy love the work so much 🤠
now that things are working though
i'm gonna ask that we try to resolve the warning (in console)
"If you need to use components with callbacks in your options, you can use 'navigation.setOptions' instead. See https://reactnavigation.org/docs/troubleshooting#i-get-the-warning-non-serializable-values-were-found-in-the-navigation-state for more details. "

it should be a well documented warning that should be easily resolved so FINAL PUSH for this sprint! You got this!